### PR TITLE
created user schema & model

### PR DESCRIPTION
**Pull request by Tyav**

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -1,7 +1,39 @@
+//Schema for User
+const mongoose = require('mongoose');
 
+//Connect to data base
+mongoose.connect('mongodb://localhost/users', { useNewUrlParser: true })
+  .then(() => console.log(`Conneted to the Database...`))
+  .catch((err) => console.log(err));
 
+const userSchema = new mongoose.Schema({
```
Comment (Nathan-E) [inline models/user.js]: nice work!!!
Please add more comments to your codebase

File: models/user.js [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+	password: {
+		type: String,
+		required: true,
+		minlength: 6,
+	},
+	email: {
+		type: String,
+		required: true,
+	},
+	private_chat: {
```
Comment (Nathan-E) [inline models/user.js]: use the right naming convention for your variable

File: models/user.js [hunk 3]
```diff
@@ -1,7 +1,39 @@
...
+		type: Object,
+		default: {},
+	},
+	group_chat: {
+		type: Object,
```
Comment (Nathan-E) [inline models/user.js]: check this link https://mongoosejs.com/docs/guide.html for the proper definition of a schema when the type is an Object

File: models/user.js (hunk 1 above)
Comment (Tyav) [reply models/user.js]: Okay thanks, would do just that

**Pull request closed without merging**